recipes/wix/src/COPYING

```diff
@@ -0,0 +1,27 @@
+This software is released under the Microsoft Reciprocal License (MS-RL) (the "License"); you may not use the software except in compliance with the License.
```

Comment:
The binary distribution used as source does not include a license file, unfortunately.